src/main/java/com/kainos/ea/backend/controllers/JobFamilyController.java

```diff
@@ -28,4 +28,16 @@ public JobFamilyController(JobFamilyService jobFamilyService) {
         return jobFamilyService.getJobFamiliesByCapabilityName(capabilityName);
     }
 
+    @PutMapping(path = "/{jobFamilyName}/{jobFamilyNewName}")
```

Comment:
I think maybe this should be a PATCH method rather than PUT. From [stackoverflow](https://stackoverflow.com/questions/31089221/what-is-the-difference-between-put-post-and-patch): 

> PUT is for checking if resource is exists then update , else create new resource
> PATCH is always for update a resource

It doesn't look like we are creating a new resource here if one doesn't exist. Correct me if I'm wrong though.

Reply:
You might be right, changed the method